pennylane/tape/tape.py

```diff
@@ -341,45 +300,12 @@ class QuantumTape(AnnotatedQueue):
     _lock = RLock()
     """threading.RLock: Used to synchronize appending to/popping from global QueueingContext."""
 
-    def __init__(self, name=None, do_queue=True):
-        super().__init__()
-        self.name = name
+    def __init__(
+        self, ops=None, measurements=None, prep=None, name=None, do_queue=True, _update=True
```

Comment:
It adds optional keyword arguments, so `QuantumTape()` still behaves the same way as before.  This did force a fix to the Lie algebra optimizer since it assumed `name` was the first argument.

If we want to start allowing either `QuantumScript` or `QuantumTape` in different areas, like `expand_tape`, we should give both objects the same call signature.